src/styling/projects-page-styling.css

```diff
@@ -1,72 +1,71 @@
 /** Multiple card view **/
 
 .pg-container {
-    padding: 0px 30px 0px;
+  padding: 0px 30px 0px;
```

Comment:
nitpick: remove unit when the value is 0

Reply:
This css is for @nupur-n's page. Nupur, could you fix this in your PR?